src/main/kotlin/calculator/Validator.kt

```diff
@@ -0,0 +1,4 @@
+package calculator
+
+val List<Double>.areAllPositive: Boolean
```

Comment:
```suggestion
val List<Double>.allPositive: Boolean
```  

이렇게 더 간단하게 네이밍해줄 수 있겠어요!

Reply:
`areAllPositive`의 경우 질문형이라 `Boolean` 값을 반환한다는 명확성이 있는 반면, `allPositive`는 모두 양수로 변환한 리스트를 반환한다고 느껴질 수도 있는 등 명확성이 부족할 수 있을 것 같은데 어떻게 생각하시나요!?

Reply:
> allPositive는 모두 양수로 변환한 리스트를 반환한다고 느껴질 수도 있는 등 명확성이 부족할 수 있을 것 같은데 어떻게 생각하시나요!?

좋습니다! 이건 개인적인 코드 스타일이기에 크게 상관 없을 것 같아요!  
저같은 경우는 Kotlin stlib 에서 `all`, `any`, `none` 와 Predicate 람다로 구성되어있기에, 간략하게 나타내는 것을 선호합니다. 😉